tests/modules/programs/neovim/plugin-config.nix

```diff
@@ -7,28 +7,27 @@ with lib;
     programs.neovim = {
       enable = true;
       extraConfig = ''
-        " This 'extraConfig' should be present in vimrc
+        let g:hmExtraConfig='%%hmExtraConfig%%'
       '';
       plugins = with pkgs.vimPlugins; [
         vim-nix
         {
           plugin = vim-commentary;
           config = ''
-            " plugin-specific config
-            autocmd FileType c setlocal commentstring=//\ %s
-            autocmd FileType c setlocal comments=://
+            let g:hmPlugins='%%hmPlugins%%'
```

Comment:
any reason this takes the form '%%hmPlugins%%' with all the `%` ?

Reply:
`%%` is sometimes used as a placeholder for stuff, also it appears in neovim paths (in backups IIRC). I would prefer something more vanilla like `HM_PLUGINS`. Other than that good to go.